src/mpvis/mpdfg/diagrammers/graphviz.py

```diff
@@ -27,6 +27,7 @@ def __init__(
         visualize_cost: bool = True,
         cost_currency: str = "USD",
         rankdir: str = "TB",
+        arc_thickness_by: str = "frequency",
```

Comment:
[REQ]
Use string literals.

```suggestion
    arc_thickness_by: Literal["frequency", "time"] = "frequency",
```